pennylane/labs/tests/phase_polynomials/test_parity_matrix.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Tests for pennylane/labs/phase_polynomials/parity_matrix.py"""
+from itertools import permutations
+
+import numpy as np
+import pytest
+
+import pennylane as qml
+from pennylane.labs.phase_polynomials import parity_matrix
+
+circ1 = qml.tape.QuantumScript(
+    [
+        qml.CNOT((3, 2)),
+        qml.CNOT((0, 2)),
+        qml.CNOT((2, 1)),
+        qml.CNOT((3, 2)),
+        qml.CNOT((3, 0)),
+        qml.CNOT((0, 2)),
+    ],
+    [],
+)
```

Comment:
The measurements argument is optional anyways :) 
```suggestion
circ1 = qml.tape.QuantumScript(
    [
        qml.CNOT((3, 2)),
        qml.CNOT((0, 2)),
        qml.CNOT((2, 1)),
        qml.CNOT((3, 2)),
        qml.CNOT((3, 0)),
        qml.CNOT((0, 2)),
    ],
)
```